test/lit.cfg

```diff
@@ -53,7 +53,9 @@ def darwin_sdk_build_version_split(version):
 
 
 def darwin_sdk_build_version_cmp(lhs, rhs):
-    return cmp(
+    def _cmp(a, b):
```

Comment:
I did put a description and a link in the commit message itself, not just the MR description, so that the message would always show up in the `git blame` of that line. (You can see it if you do a `git show 389d9e581bd480fb0444633682a7baf985a02df8`)

I'll see what I can do to condense that and transfer it to a code comment. Thank you for the review.